### PR TITLE
#1594 Correct unit tests

### LINKED ISSUE #1594: Unit tests for CPP models are incorrect
Sequence of events incorrect

### PR DESCRIPTION
**Pull request by lecourtoisflo**
closes #1594

### CODE REVIEW

File: dynawo/sources/Models/CPP/ModelNetwork/test/TestLoad.cpp [hunk 1]
```diff
@@ -568,16 +571,14 @@ TEST(ModelsModelNetwork, ModelNetworkLoadJt) {
...
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[1], -153.29392144688848);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[2], -87.596526541079143);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[3],  8.4895038000592429);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[4],  2.3388791341676249);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[5],  1.3365023623815);
+  ASSERT_EQ(smj.nbElem(), 4);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[0], 2);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[1], 8.4895038000592429);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[2], 5.2397892818982124);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[3], 2.9941653039418354);
```
Comment (rosiereflo) [inline dynawo/sources/Models/CPP/ModelNetwork/test/TestLoad.cpp:578]: Aren't we missing 2 indexes here?

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/CPP/ModelNetwork/test/TestLoad.cpp (hunk 1 above)
Comment (lecourtoisflo) [reply dynawo/sources/Models/CPP/ModelNetwork/test/TestLoad.cpp:578]: No, the error in the test make a threshold be not passed so we have less 2 terms

Comment (lecourtoisflo) [reply dynawo/sources/Models/CPP/ModelNetwork/test/TestLoad.cpp:578]: Are we sure about that ? seems to me that it was to test with the correct inputs

Comment (lecourtoisflo) [reply dynawo/sources/Models/CPP/ModelNetwork/test/TestLoad.cpp:578]: OK

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**